### DISCUSSION TITLE
Video snaps for Koriki

### DISCUSSION THREAD

**Discussion by ThOoM013** [rg35xx-cfw/Koriki#55] [Q&A] [answered]
Hi my name is Thomas first of I would like to thank everyone involved in this wonderful project. In now using batocera on my 35xx but i read that the support for future updates has stopped and that koriki will be the new focus for development. What I really really enjoy on batocera are the options for video snaps. It gives you a better understanding of what type of game you are playing vs a screenshot or box art. Is this option possibly added to Koriki in the future? Is this even possible?

Thank you in advance for replying. And to all the developers... Good luck!

**Comment by acmeplus** [comment, marked as answer]
Sorry for the late reply. One of the reasons I decided to create a koriki/batocera hybrid firmware was because of the memory limitations on the RG35XX. It only has 256MB and even with virtual memory enabled (swap) it does not perform really well when you hit the 256MB memory mark. The video thumbnails in batocera take a toll with the memory, so that (among other reasons) was one of the key factor to use simplemenu (soon to be replaced with simplermenu+).

Adding video previews is something that will be considered for simplermenu+, but it's still early for that project. It's also unknown whether they will perform correctly in the rg53xx. In any case, adding video previews should be feasible and will eventually be explored.

**Reply by ThOoM013** [reply]
Thank you for you're explanation. Makes total sence.  Waiting patiently on the 1.0 release.

**Comment by job1022** [comment]
it would be beatiful to add it.
